### PR TITLE
FAQ Contact diary

### PR DESCRIPTION
**Pull request by dsarkar**
Closes https://github.com/corona-warn-app/cwa-website/issues/784

### CODE REVIEW

**Comment by dsarkar** [comment]
EN
image

**Comment by dsarkar** [comment]
DE

image

**Review by loki-cui — APPROVED** [review]

File: src/data/faq.json [hunk 1]
```diff
@@ -778,7 +778,17 @@
...
                         ]
-                    }
+                    },
+                    {
+                        "title": "Contact Diary",
+                        "anchor": "contact_diary",
+                        "active": true,
+                        "textblock": [
+                            "Since release of the contact diary function, we continuously received suggestions for changes and improvements. All those ideas will be reviewed technically and together with the client regarding:",
+                            "<ul><li>Benefit for the consumers of the diary (e.g., the health authorities)</li><li>User friendliness of the UI,</li><li>extension by functions (e.g. recording of additional parameters),</li><li>technical feasibility (or the effort required for implementation), and</li><li>legal framework and limits</li></ul>",
```
Comment (MikeMcC399) [inline src/data/faq.json:788]: remove comma after e.g.

(e.g., the health authorities) => (e.g. the health authorities)

Capitalise each entry in the list to be consistent:
extension => Extension 
technical => Technical 
legal => Legal

File: src/data/faq_de.json [hunk 2]
```diff
@@ -787,6 +787,16 @@
...
                             "Die Details dazu finden Sie (in englischer Sprache) in unserem <a href='https://github.com/corona-warn-app/cwa-documentation/blob/master/solution_architecture.md' target='_blank' rel='noopener noreferrer'>Solution-Architecture-Dokument</a>."
                         ]
+                    },
+                    {
+                        "title": "Kontakt-Tagebuch",
+                        "anchor": "contact_diary",
+                        "active": true,
+                        "textblock": [
+                            "Seit Einführung des Kontakt-Tagebuchs erhalten wir kontinuierlich Änderungs- und Verbesserungsvorschläge. Alle diese Ideen prüfen wir technisch und gemeinsam mit dem Auftraggeber bezüglich:",
+                            "<ul><li> Nutzen für die Konsument*innen des Tagesbuchs (z.B. die Gesundheitsämter)</li><li> Anwenderfreundlichkeit ders UI,</li><li>Erweiterung durch Funktionen (z.B. Erfassen zusätzlicher Parameter),</li><li>technische Umsetzbarkeit (bzw. die Aufwände für die Implementierung) sowie</li><li>rechtlicher Rahmenbedingungen und Grenzen</li></ul>",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:797]: Typo: ders 
in "Anwenderfreundlichkeit ders UI"

Ist das grammatikalisch richtig?
bezüglich: ... technische Umsetzbarkeit
oder "technischer"

**Review by MikeMcC399 — COMMENTED** [review]
I was too late to catch the merge, but please look anyway.

File: src/data/faq_de.json (hunk 2 above)
Comment (Ein-Tim) [reply src/data/faq_de.json:797]: "technischer" would be correct

**Pull request merged**